pkg/i18n/english.go

```diff
@@ -1278,6 +1279,7 @@ func EnglishTranslationSet() *TranslationSet {
 		GlobalTitle:                          "Global keybindings",
 		ConflictsResolved:                    "All merge conflicts resolved. Continue?",
 		Continue:                             "Continue",
+		UnstagedFilesAfterConflictsResolved:  "Files have been modified after staging the resolved conflicts. Auto-stage them and continue?",
```

Comment:
```suggestion
		UnstagedFilesAfterConflictsResolved:  "Files have been modified since conflicts were resolved. Auto-stage them and continue?",
```
This reads a little better to me (though can't explain why)

Reply:
Fine with me, changed.